doc/programming_guide/asrc/overview.rst

```diff
@@ -2,11 +2,32 @@
 Overview
 ********
 
+.. warning::
+
+   This example is based on the RTOS framework and drivers. This choice allowed simplifying the example design, but it lead to some latency in the system.
+   The main sources of latency are:
+
+      - Large block size used for ASRC processing: this is necessary to minimise latency associated with the intertile context and thread switching overhead.
+      - Large size of the buffer to which the ASRC output samples are written: a stable level (half full) must be reached before starting streaming out over USB.
+      - RTOS task scheduling overhead between the tasks.
+      - bInterval of USB in the RTOS drivers is set to 4, i.e. one frame every 1ms.
+      - Block based implementation of the USB and I2S RTOS drivers.
+
+   The expected latencies for USB at 48 kHz are as follows:
+
+      - USB -> ASRC -> I2S: from 8 ms at |I2S| at 192 kHz to 22 ms at 44.1 kHz
+      - I2S -> ASRC -> USB: from 13 ms at |I2S| at 192 kHz to 19 ms at 44.1 kHz
```

Comment:
`from 13 ms at |I2S|` -> `from 13 ms with |I2S|`

Reply:
Fixed in https://github.com/xmos/sln_voice/pull/365/commits/86b87358a4a8763ecd3426194ce91bcc9141c5fa.